qtconsole/console_widget.py

```diff
@@ -2188,6 +2188,23 @@ def _insert_plain_text(self, cursor, text, flush=False):
                             cursor.select(QtGui.QTextCursor.Document)
                             cursor.removeSelectedText()
 
+                    elif act.action == 'move' and act.unit == 'line':
+                        if act.dir == 'up':
+                            for i in range(act.count):
+                                cursor.movePosition(
+                                    QtGui.QTextCursor.Up)
```

Comment:
```suggestion
                                    QtGui.QTextCursor.Up
                                )
```

Improve code style a bit.